src/functions.js

```diff
@@ -626,19 +626,117 @@ defineFunction([
 defineFunction([
     "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
     "\\check", "\\hat", "\\vec", "\\dot",
-    // We don't support expanding accents yet
-    // "\\widetilde", "\\widehat"
+    "\\widehat", "\\widetilde", "\\overrightarrow", "\\overleftarrow",
+    "\\Overrightarrow", "\\overleftrightarrow", "\\overgroup",
+    "\\overlinesegment", "\\overleftharpoon", "\\overrightharpoon",
 ], {
     numArgs: 1,
 }, function(context, args) {
     const base = args[0];
+
+    const isStretchy = !utils.contains([
+        "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+        "\\check", "\\hat", "\\vec", "\\dot",
+    ], context.funcName);
+
+    const isShifty = !isStretchy || utils.contains([
+        "\\widehat", "\\widetilde", "\\overrightarrow", "\\overrightharpoon",
+    ], context.funcName);
+
     return {
         type: "accent",
-        accent: context.funcName,
+        label: context.funcName,
+        isStretchy: isStretchy,
+        isShifty: isShifty,
+        value: ordargument(base),
         base: base,
     };
 });
 
+// Horizontal stretchy braces and brackets
+defineFunction([
+    "\\overbrace", "\\overbracket", "\\underbrace", "\\underbracket",
+], {
+    numArgs: 1,
+    numOptionalArgs: 2,
+    argTypes: ["size", "size", "original"],
+}, function(context, args) {
+    const thickness = args[0];
+    const height = args[1];
+    const base = args[2];
```

Comment:
I see we're being inconsistent with `base` and `body`.  We should clean that up at some point.  Don't worry about it in this PR since it's big enough as is.